src/doc/rustc/src/platform-support/wasm32-unknown-emscripten.md

```diff
@@ -0,0 +1,109 @@
+# `wasm32-unknown-emscipten`
+
+**Tier: 2**
+
+The `wasm32-unknown-emscripten` target is a WebAssembly compilation target which
+uses the [Emscripten](https://emscripten.org/) compiler toolchain. Unlike the
+[`wasm32-unknown-unknown`](./wasm32-unknown-unknown.md) target, which only has
+partial support for the Rust standard library, `wasm32-unknown-emscripten`
+integrates with JavaScipt on the web to provide a more featureful standard library.
+This target is often used when compiling for the web, accessing web APIs, or
+interoperating with C and C++ code.
+
+One existing user of this target is the
+[`pyodide` project](https://pyodide.org/) which provides a Python runtime in
+WebAssembly using Emscripten and compiles Python extension modules written in Rust
+to the `wasm32-unknown-emscripten` target.
+
+If you want to target the web but no not need interop with C/C++ or you want to
+produce a bare-bones WebAssembly binary and you do not require access to the host
+environment (e.g. stdio, file access, networking, etc.), consider using the
+[`wasm32-unknown-unknown`](./wasm32-unknown-unknown.md) target instead.
+
+If you need access to the host environment, consider using one of the WASI targets
+([`wasm32-wasip1`](./wasm32-wasip1.md) or [`wasm32-wasip2`](./wasm32-wasip2.md)).
+
+## Target maintainers
+
+When this target was added to the compiler, platform-specific documentation here
+was not maintained at that time. This means that the list below is not
+exhaustive, and there are more interested parties in this target. That being
+said, those interested in maintaining this target are:
```

Comment:
I realize this is probably copy/pasted from the `wasm32-unknown-unknown` target but given the set of three maintainers below you can probably drop this clause. I mostly added it to the unknown target because I was the only one listed and wanted to clarify why there was just one.

Reply:
Done